src/data/callouts.ts

```diff
@@ -26,8 +26,17 @@ export interface InputCalloutComponentSchema
     | "number"
     | "password"
     | "textfield"
-    | "textarea";
+    | "textarea"
+    | "content"
+    | "phoneNumber"
+    | "currency"
+    | "datetime"
+    | "time"
+    | "url"
+    | "signature";
   input: true;
+  filePattern?: string;
```

Comment:
Is this just for a specific component type? If so it would be better to add a separate interface, similar to the select and radio components below

Reply:
es for file, I will create a new interface